### ISSUE TITLE
Google Chrome focuses window a lot

### ISSUE THREAD

**Issue by Sanjo** [xolvio/meteor-cucumber#142]
When I change a save a @dev spec the Chrome window (where the tests run in) focuses multiple times. This makes it hard to continue writing the spec.

I found out that when a screenshot is taken, Chrome takes the focus.

__Workaround:__ Disable taking screenshots on error. You can do this by starting your app with this environment variable set to:
```
export CHIMP_OPTIONS="--tags=@dev --browser=chrome --no-screenshotsOnError"
```

**Comment by workflow** [comment]
+1

**Comment by Sanjo** [comment]
I found out that taking screenshots will focus Chrome. I've added a workaround in my initial post. This workaround will work with the upcoming release (>= 0.9.4).

**Comment by samhatoum** [comment]
https://github.com/xolvio/chimp/pull/29

Reminder to enable this by default when CI mode is detected also for any browser

**Comment by Sanjo** [comment]
Will be fixed in >= 0.10.0. It is released later today.
